src/main/js/helpers/axis.js

```diff
@@ -191,36 +197,139 @@ const getAxesScale = (axis, scale, config) => {
         getAxisInfoOrientation(config.axis.x.orientation)
     );
 
-    // If ticksCount is undefined or greater than TICKS_MAXCOUNT
-    // AND if the Y2 is visible, then utilize a default value for
-    // the ticksCount. This is based on the ranges of the Y & Y2 axes.
+    if (hasY2Axis(config.axis)) {
+        // Y and Y2 axes - custom tick values. Takes priority
+        // and ignores ticksCount if it is set. Will not work if only
+        // Y2 ticks are provided.
+        if (utils.isDefined(config.axis.y.ticks.values)) {
```

Comment:
Probably, Is it safe to verify y2-ticks as well utils.isDefined(config.axis.y2.ticks.values)?

What if the consumer only provided y-ticks values and y-2 ticks values are not defined?

Reply:
If only Y ticks are provided for a 2 axis graph, then the Y2 ticks will be automatic, as shown below:

<img width="885" alt="image" src="https://user-images.githubusercontent.com/38024451/86389297-b8669d80-bc5b-11ea-9d50-db712b1eb87d.png">

This decision was deliberate so that consumers can have a starting point for determine what Y2 values should be used to maintain alignment. There is also the case where the automatic values are aligned if both axes have the same range. In this example, the Y ticks are custom while the Y2 ticks are automatic:
<img width="945" alt="image" src="https://user-images.githubusercontent.com/38024451/86389849-9ae60380-bc5c-11ea-8372-88caab3d9d29.png">